src/common/db/schemas.lua

```diff
@@ -0,0 +1,20 @@
+local schemas = {}
+
+schemas.ant_index = [[
+        CREATE TABLE IF NOT EXISTS ant_index (
+            ant_id TEXT PRIMARY KEY NOT NULL, 
+            owner TEXT NOT NULL,
+            registered_at INTEGER NOT NULL
```

Comment:
what is this representing? can we add `last_updated` as well?

Reply:
Sure, this was just a utility piece - only gets added at registration and uses msg.Timestamp to denote the registration time, we can definitely add the last_updated column to this.